rawdisk/filesystems/detector.py

```diff
@@ -73,7 +73,10 @@ def detect_mbr(self, filename, offset, fs_id, verbose=False):
             plugins = self.mbr_plugins.get(fs_id)
             for plugin in plugins:
                 if plugin.detect(filename, offset):
-                    return plugin.get_volume_object()
+                    volume = plugin.get_volume_object()
+                    if verbose and not volume:
+                        print "Error: failed to get volume"
```

Comment:
This should not be marked as an error, simply fs plugin was not matched and should just be ignored